src/lib/pg-bark/_impls/alternative-add-appointment.ts

```diff
@@ -0,0 +1,50 @@
+import { Err, Ok, Result } from "@/lib/utilities/result";
+import { CODE } from "@/lib/utilities/bark-code";
+import { PgBarkServiceConfig } from "../pg-bark-service";
+import { dbBegin, dbCommit, dbRelease, dbRollback } from "@/lib/data/db-utils";
+import { selectAppointmentStats } from "../_queries/select-appointment-stats";
+import { insertAppointment } from "../_queries/insert-appointment";
+
+export async function addAppointment(
```

Comment:
Given that—with this new query approach—these are relatively concise, perhaps they can be implemented directly as class methods. WDYT?

Reply:
Bad idea. The class would become fat with code, some would be unrelated to one another. I created a lib/bark/operations folder instead for operation functions.